src/phsoar_null_router/soar_null_router_connector.py

```diff
@@ -21,6 +21,9 @@
 import json
 from bs4 import BeautifulSoup
 
+__version__ = 'GITHUB_TAG'
+__git_hash__ = 'GITHUB_SHA'
+__deployed__ = 'GITHUB_DEPLOYED'
 
```

Comment:
The error seems to indicate that this requires a second blank line here

Reply:
Good call. Done.

Reply:
```suggestion


```